.golangci.yml

```diff
@@ -150,3 +151,10 @@ linters:
 #    - wsl # Whitespace Linter - Forces you to use empty lines! [fast: true, auto-fix: false]
 #    - varcheck # Finds unused global variables and constants [fast: false, auto-fix: false]
 #    - ifshort # Checks that your code uses short syntax for if-statements whenever possible [fast: true, auto-fix: false]
+issues:
+  exclude-rules:
+    - linters:
+        - gocritic
+      text: "^builtinShadow: shadowing of predeclared identifier: (min|max)"
```

Comment:
added because errors like: 
```
 Error: admin/model_index_options.go:65:6: builtinShadow: shadowing of predeclared identifier: max (gocritic)
  	var max int = 180
  	    ^
```